resources/views/emails/angeltype-added.twig

```diff
@@ -0,0 +1,11 @@
+{% extends "emails/mail.twig" %}
```

Comment:
Which action triggers this e-mail?

Reply:
This email is triggered when an angel gets added to an angeltype by a ShiCo